button-merchant/src/main/java/com/usebutton/merchant/ButtonInternalImpl.java

```diff
@@ -68,15 +66,6 @@
     }
 
     public void configure(ButtonRepository buttonRepository, String applicationId) {
-        final boolean isApplicationIdValid = ButtonUtil.isApplicationIdValid(applicationId);
-        if (!isApplicationIdValid) {
```

Comment:
Is it worth keeping this check for a valid app id? It could possibly save partners some headaches.